src/scripts/settings/config.js

```diff
@@ -0,0 +1,159 @@
+(function () {
```

Comment:
Should this file be named something like cc-config.js since it seems to be all Concept Coach related. Just to make it clear to future devs and interns.